data/homepage.yaml

```diff
@@ -7,10 +7,20 @@ banner:
   redhat_logo_alt: Red Hat logo
   welcome_alt: Welcome to Ansible
   platform_learn: Learn about Red Hat Ansible Automation Platform
+events:
+  section_icon: /assets/images/calendar-cyan.svg
+  icon_alt: Ansible community event icon
+  title: Events
+  intro: 
+  url: 
```

Comment:
It seems like the Events banner has been removed. Should we not just delete these fields if they are unused?

Reply:
That is up to you as I thought it would be good to have in the event the banner is brought back.

Reply:
Gotcha. This is why atomic commits - isolated changes - are preferred. In order to bring the event banner back it would be much easier to revert a single commit than to reconstruct from history.

In any case if we want to drop the event banner from the homepage, I would suggest simply removing the include directive from the main homepage template.